### PR TITLE
Fix unintentional readme reference

### PR DESCRIPTION
**Pull request by grevend**
This pull request furthermore streamlines the readme description and formatting.

### CODE REVIEW

File: NOTICE.md [hunk 1]
```diff
@@ -12,3 +12,7 @@ file in the Carbyne Stack
 ### Robert Bosch GmbH
 
 - Greven David <fixed-term.David.Greven@de.bosch.com>
+
+### Unassociated
```
Comment (strieflin) [inline NOTICE.md]: Please rename to _Non-Affiliated_

File: README.md [hunk 2]
```diff
@@ -5,16 +5,16 @@
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)
 
-This library provides a set of common types and testing utilities for all
-Carbyne Stack services, clients and other components. The content is split into
-two distinct dependencies. Common-types for public API signature types, as well
-as other shared code and common testing utilities which are needed to implement
-tests in Carbyne Stack in a desirable manner with JUnit argument sources and
-AssertJ custom assertion.
+This library provides a set of shared types and testing utilities for all
```
Comment (strieflin) [inline README.md]: Have you been running pre-commit? Dangling spaces should have been detected.

**Review by strieflin — CHANGES_REQUESTED** [review]
Hi @grevend, great to have you back as a contributor!

File: NOTICE.md (hunk 1 above)
Comment (grevend) [reply NOTICE.md]: Done

File: README.md (hunk 2 above)
Comment (grevend) [reply README.md]: Fixed

**Review by strieflin — APPROVED** [review]
LGTM

**Pull request merged**